src/Services/DocumentSign/SignRequest.php

```diff
@@ -173,7 +177,8 @@ public function signShow(string $id): array
 
             return [
                 'id' => Arr::get($response, 'uuid', $id),
-                'status' => Arr::get($response, 'status')
+                'status' => Arr::get($response, 'status'),
+                'signed_on' => Arr::get($response, 'signers.1.signed_on')
```

Comment:
return all signers  as an array change name to signed_at
return email and signed_on as signed_at